app/commands/mentor/update_stats.rb

```diff
@@ -3,28 +3,26 @@ class Mentor::UpdateStats
 
   queue_as :default
 
-  initialize_with :mentor,
-    update_counts: false,
-    update_satisfaction_rating: false
+  initialize_with :mentor, :track
 
   # This command:
   # - Gets the old values
   # - Sets the new values
   # - Triggers some extra checks if they've changed
   def call
-    # Actively use the cache to get these values as we don't want
-    # to follow the automatic setting pathway we normally use.
-    old_values = [
-      mentor.data.cache['num_students_mentored'],
-      mentor.data.cache['num_solutions_mentored'],
-      mentor.data.cache['mentor_satisfaction_percentage']
-    ]
-    new_values = old_values.clone
```

Comment:
We don't bother with the whole old vs new thing any more. It was necessary before as this command got called all the time, but now it's only called at specific relevant times, it's not worth the hassle.